addons/web/static/src/views/kanban/kanban_controller.js

```diff
@@ -25,7 +25,7 @@ export class KanbanController extends Component {
     setup() {
         this.actionService = useService("action");
         const { Model, archInfo } = this.props;
-        this.model = useModel(Model, this.modelParams);
+        this.model = useModel(Model, this.modelParams, this.modelOptions);
```

Comment:
not sure about that. can't you just subclass the kanbancontroller to add your info in modelParams? maybe @aab-odoo will have an opinion

Reply:
This diff is there to provide the `onWillStartAfterLoad`. I can move it to `modelParams` if you want but since `useModel` already has a third `options` parameter already allowing a custom `onWillStart`, that way looks more logical to me